chameleon-system/chameleon-base
# 332 add core ui based theme with build process

| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? |no
| Tests pass?   | unknown
| Fixed issues  | chameleon-system/chameleon-system#332
| License       | MIT

This PR contains everything to...
1. re-install CoreUI
2. build CourUI using CoreUI's own build process
3. store scss variables to override CoreUI's styles
4. add custom scss as imports
5. build CoreUI with all overrides and additional scss files
6. develop using a watch mode 

See `customer/vendor/chameleon-system/chameleon-base/src/CoreBundle/Resources/Theme/readme.md` for more details.
